app/workers/rapid_ready_for_decision/form526_asthma_job.rb

```diff
@@ -32,7 +32,17 @@ def query_and_assess_lighthouse(form526_submission)
     def assess_medications(medications)
       return [] if medications.blank?
 
-      RapidReadyForDecision::LighthouseMedicationRequestData.new(medications).transform
+      asthma_keywords = RapidReadyForDecision::Constants::DISABILITIES[:asthma][:keywords]
```

Comment:
I'd change this into a constant: `ASTHMA_KEYWORDS`

Reply:
@yoomlam interesting, that throws a syntax error: 

>  dynamic constant assignment

it's not dynamic in the way i'm thinking of the word...

Reply:
Ok. Nevermind then. I guess a lookup in a hash (even though it's a frozen hash) is considered dynamic 🤷